Lib/sqlite3/test/regression.py

```diff
@@ -188,6 +188,7 @@ def __init__(self, con):
         cur = Cursor(con)
         with self.assertRaises(sqlite.ProgrammingError):
             cur.execute("select 4+5").fetchall()
+        self.assertRaises(sqlite.ProgrammingError, cur.close)
```

Comment:
`ProgrammingError` is raised by a lot of things so I wonder if we should add a test for the exception message to make it more future-proof.

Reply:
All right. Should i also change the `assertRaises()` that was already in `CheckCursorConstructorCallCheck()`?

Reply:
I'd say let's do it only for new additions to the test.